specification/synapse/data-plane/Microsoft.Synapse/preview/2021-06-01-preview/runNotebook.json

```diff
@@ -0,0 +1,359 @@
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "./artifacts.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/runNotebookApi/versions/{version}/run/{runId}/snapshot": {
+      "get": {
+        "tags": [
+          "RunNotebook"
+        ],
+        "operationId": "Get_RunNotebookSnapshot",
+        "x-ms-examples": {
+          "Get Run Notebook Snapshot": {
+            "$ref": "./examples/RunNotebook_GetSnapshot.json"
+          }
+        },
+        "description": "Get RunNotebook Snapshot for run id.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/VersionParameter"
+          },
+          {
+            "$ref": "#/parameters/RunId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok.",
+            "schema": {
+              "$ref": "#/definitions/RunNotebookResponse"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "./artifacts.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Error": {
+      "description": "Run notebook error.",
+      "type": "object",
+      "properties": {
+        "ename": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Error name."
+        },
+        "evalue": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Error message."
+        },
+        "traceback": {
+          "description": "Error trace.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Result": {
+      "description": "Result of run notebook.",
+      "type": "object",
+      "properties": {
+        "runId": {
+          "type": "string",
+          "description": "Run id."
+        },
+        "runStatus": {
+          "description": "Status of the run notebook.",
+          "type": "string"
+        },
+        "lastCheckedOn": {
+          "description": "Timestamp of last update.",
+          "type": "string"
+        },
+        "sessionId": {
+          "description": "Livy session id.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "sparkPool": {
+          "description": "SparkPool name.",
+          "type": "string"
+        },
+        "snapshot": {
+          "description": "Notebook output.",
+          "type": "object"
+        },
+        "sessionDetail": {
+          "description": "Run notebook session details.",
+          "type": "object"
+        },
+        "exitValue": {
+          "description": "Output of exit command.",
+          "type": "string"
+        },
+        "errors": {
```

Comment:
It's better to expose `error` instead of `errors`. It doesn't make sense that `errors` is not an array. I have marked `errors` as obsolete.